server/routes/mention.js

```diff
@@ -21,10 +25,16 @@ router.get("/:id", async (req, res) => {
   }
 });
 
-router.get("/:companyName", async (req, res) => {
+router.get("/company/:companyName", async (req, res) => {
   try {
     const { companyName } = req.params;
-    const result = await Mention.find({ companyName });
+    const contentResult = await Mention.find({
```

Comment:
probably want to combine the two regExp rather than doing one by one than concat because it is faster